databases/week2/more-queries.sql

```diff
@@ -0,0 +1,24 @@
+USE hyf;
+select * from task;
+
+-- 1 
+SELECT  user.name, user.email FROM user
+WHERE user.email REGEXP '@spotify.com$';
+
+-- 2
+SELECT * FROM task
+JOIN user 
+ON task.id = user.id
+JOIN status
+ON task.id = task.status_id
+WHERE user.name REGEXP 'Donald Duck' AND status.name = 'Not started' ;
+
+-- 3
+SELECT * FROM task
+JOIN user
+ON task.id = user.id
+WHERE user.name REGEXP 'Maryrose Meadows' AND month(created) = 09 ;
+
+-- 4
+SELECT COUNT(*) as october FROM task
+GROUP BY month(created) = 10;
```

Comment:
When you use the group by there is no need to use the " = 10", you're just creating groups, in this case you create groups of tasks based on the month when they were created.
 Also, it would be nice to also select the month(created) to see how many tasks were created in which month :). 
 Again, let me know if something is not clear :)